docs/src/pages/components/about-the-lab/about-the-lab.md

```diff
@@ -43,12 +43,12 @@ yarn add @material-ui/core@next
 In order to benefit from the [CSS overrides](/customization/globals/#css) and [default prop customization](/customization/globals/#default-props) with the theme, TypeScript users need to import the following types. Internally, it uses [module augmentation](/guides/typescript/#customization-of-theme) to extend the default theme structure with the extension components available in the lab.
 
 ```tsx
-import type '@material-ui/lab/themeAugmentation';
+import '@material-ui/lab/themeAugmentation';
```

Comment:
I can suggest to use such variant:
```
// noinspection ES6UnusedImports
import type {} from '@material-ui/lab/themeAugmentation';
```

It will allow us to remove synthetic index.js, and avoid emitting unnecessary code to the runtime.

Reply:
Is `// noinspection ES6UnusedImports` required?

Reply:
import type comes with TypeScript [3.8](https://www.typescriptlang.org/docs/handbook/modules.html#importing-types). So far, we support TypeScript down to 3.5.

Reply:
It depends on IDE configuration. Webstorm shows an error without that comment.

So, it is just an example of code in docs. I believe we can do smth like that:

> ```
> // noinspection ES6UnusedImports
> import type {} from '@material-ui/lab/themeAugmentation';
> ```
> or if you use TS older than 3.8
> ```
> import '@material-ui/lab/themeAugmentation';
> ```

Reply:
Ok, we will keep `import '@material-ui/lab/themeAugmentation';` for now, the overhead is marginal.